games/lockpicking/src/js/components/Tabs.js

```diff
@@ -0,0 +1,80 @@
+import { uniqueId } from "../utils/uniqueId.js";
+
+/**
+ * Creates a new Tabs
+ * @class Tabs
+ */
+
+export class Tabs {
+  constructor(options) {
+    this.node = null;
+    this.options = options;
+    this.id = "";
+  }
+
+  getHTML() {
+    return `
+      ${this.renderInputs()}
+      <div class="tabs">
+      ${this.renderLabels()}
+       
+      </div>
+      <div class="tabs__content">
+        ${this.renderContent()}
+      </div>
+    `;
+  }
+
+  renderInputs() {
+    const tabKeys = Object.keys(this.options.tabs);
+    let HTML = "";
+    tabKeys.forEach((tabKey) => {
+      HTML += `<input type="radio" id="${this.options.tabs[tabKey].id}" name="tabs-radio" class="tabs__radio visually-hidden" />`;
+    });
+    return HTML;
+  }
+  //return string
+  renderLabels() {
+    const tabKeys = Object.keys(this.options.tabs);
```

Comment:
methods:
- renderLabels
- renderContent

Is pretty similar. Try to refactor it.